geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/Dockerfile_tmpl

```diff
@@ -26,8 +26,8 @@ RUN \
 
 WORKDIR /app
 COPY . /app
-COPY --from=builder /usr/lib/node_modules/ngeo/dist/* \
-    /etc/static-ngeo/
+COPY --from=builder /app/{{package}}_geoportal/locale/ /app/{{package}}_geoportal/locale/
```

Comment:
mmm you copy the same content at the same place ?

Reply:
Yes, but from builder to runner :-)